script/Ciel/Day 3/03_01_CIEL03_2_3.txt

```diff
@@ -623,5 +623,5 @@ Okay then, bye-bye! To the cafeteria you go, hurry up or lunch break will be ove
 [sha:0e16019fbb14746f88e698aea00e8b3b3a0bb713]{
 -- Page 104, Offset 5098.
 -- かつん、かつん、と<堅い靴|パンプス>の音が部室棟の奥へと消えていった。
-%{i}Pitter patter pitter patter%{/i}. The sound of her hard shoes disappears into the back of the club building.
+%{i}Pitter-patter, pitter-patter%{/i}. The sound of her hard shoes disappears into the back of the club building.
```

Comment:
mm, i kinda like the old one but i'll let ross weigh in because i'm kinda unsure

Reply:
Original feels more brisk without the punct, I kinda prefer it that way